addon/webextension/background/loadSelector.js

```diff
@@ -25,19 +25,19 @@ window.loadSelector = (function () {
     let lastPromise = Promise.resolve(null);
     scripts.forEach((script) => {
       lastPromise = lastPromise.then(() => {
-        return chrome.tabs.executeScript({
+        return browser.tabs.executeScript({
           file: script
         }, () => {
-          if (chrome.runtime.lastError
-              && chrome.runtime.lastError.message != "Script returned non-structured-clonable data") {
-              catcher.unhandled(chrome.runtime.lastError, {script: script});
-              console.error("Error loading script", script, ":", chrome.runtime.lastError);
+          if (browser.runtime.lastError
+              && browser.runtime.lastError.message != "Script returned non-structured-clonable data") {
+              catcher.unhandled(browser.runtime.lastError, {script: script});
+              console.error("Error loading script", script, ":", browser.runtime.lastError);
```

Comment:
This particular error shows up in the console, but doesn't show up as a rejected promise.  It's a nuisance error.

AFAIU Promise.all doesn't ensure that the promises complete in order.  That is, the first module should be completely loaded before attempting the load the second module.